pkg/querier/querier.go

```diff
@@ -60,6 +62,7 @@ var (
 
 // RegisterFlags adds the flags required to config this to the given FlagSet.
 func (cfg *Config) RegisterFlags(f *flag.FlagSet) {
+	cfg.StoreGatewayGRPCClientConfig.RegisterFlagsWithPrefix("querier.store-gateway-client", f)
```

Comment:
The store-gateway is experimental so any of its flags should be prefixed with "experimental".

```suggestion
	cfg.StoreGatewayGRPCClientConfig.RegisterFlagsWithPrefix("experimental.querier.store-gateway-client", f)
```